src/main/java/org/broadinstitute/dsm/shipping/UPSTracker.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dsm.shipping;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.broadinstitute.dsm.model.ups.UPSTrackingResponse;
+import org.broadinstitute.dsm.util.DDPRequestUtil;
+import org.broadinstitute.dsm.util.NanoIdUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class UPSTracker {
+
+    private static final Logger logger = LoggerFactory.getLogger(UPSTracker.class);
+
+    private final String baseEndpoint;
+    private final String username;
+    private final String password;
+    private final String accessKey;
+
+    public UPSTracker(String baseEndpoint,
+                      String username,
+                      String password,
+                      String accessKey) {
+
+        this.baseEndpoint = baseEndpoint;
+        this.username = username;
+        this.password = password;
+        this.accessKey = accessKey;
+    }
+
+    public UPSTrackingResponse lookupTrackingInfo(String trackingId) {
```

Comment:
Factored this out into its own class so that we can inspect various kit histories during development.